ImageGallery/src/org/sleuthkit/autopsy/imagegallery/ImageGalleryController.java

```diff
@@ -437,48 +438,57 @@ public synchronized void reset() {
      * @return true if datasources table is stale
      */
     boolean isDataSourcesTableStale() {
+        return (getStaleDataSourceIds().isEmpty() == false);
+    }
```

Comment:
I have started using apache CollectionUtils.isNotEmpty() to avoid the == false and the codacy warning it often generates.